src/core/wordle/api.js

```diff
@@ -0,0 +1,92 @@
+// Database access operations
+const path = require('path');
+const JSONdb = require('simple-json-db');
+
+/**
+ * @return {JSONdb}
+ */
+function initDB() {
+  const file = path.resolve(__dirname, 'db.json');
+  return new JSONdb(file);
+}
+
+/**

```

Comment:
In my humble opinion, the commands that interact with database should exist within `utils/database` folder which already exists.